### DISCUSSION TITLE
How to fatch relations from SupabaseBD on budibase app?

### DISCUSSION THREAD

**Discussion by ahmadblueeast** [Budibase/budibase#6919] [Help]
I create a DB(Supabase) and then I made tables on Supabase PostgreSQL DB and build their relations. Now we connect DB to budibase app.

Now we get data from DB but when we fetch data from DB. It Creates connections but it would not create relations between tables. Also, change column dataType. 
![Screenshot from 2022-07-27 15-47-54](https://user-images.githubusercontent.com/95227479/181230558-fccf0667-2165-4471-9db1-14d071bf56d9.png)
![Screenshot from 2022-07-27 15-43-28](https://user-images.githubusercontent.com/95227479/181230571-40a84f46-c463-4fc6-ad2b-a54ea664cee2.png)

**Comment by melohagan** [comment]
Hey @ahmadblueeast 

You need to **define relationships**. There's a Postgres example here: https://docs.budibase.com/docs/build-a-crud-app-sql#using-relationships-between-sql-tables
